creativecloud/scripts/utils.js

```diff
@@ -38,3 +38,49 @@ export const [setLibs, getLibs] = (() => {
     }, () => libs,
   ];
 })();
+
+function getDecorateAreaFn() {
+  let lcpImgSet = false;
+
+  // Load LCP image immediately
+  const eagerLoad = (lcpImg) => {
+    lcpImg?.setAttribute('loading', 'eager');
+    lcpImg?.setAttribute('fetchpriority', 'high');
+    if (lcpImg) lcpImgSet = true;
+  };
+
+  async function loadLCPImage(area = document, { fragmentLink = null } = {}) {
+    const firstBlock = area.querySelector('body > main > div > div');
+    let fgDivs = null;
```

Comment:
i would go for 'fragmentDivs' here, imho 'fg' shortcut is not clear enough

Reply:
This is pre-existing code but I think it is not related to fragments tbh. Maybe foreground? It's related to an `interactive-marquee` block, see https://github.com/adobecom/cc/blob/main/creativecloud/blocks/interactive-marquee.